packages/office-ui-fabric-react/src/components/GroupedList/GroupShowAll.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import {
+  BaseComponent,
+  autobind,
+  css
+} from '../../Utilities';
+import { Link } from '../../Link';
+import { IGroupDividerProps } from './GroupedList.Props';
+import { GroupSpacer } from './GroupSpacer';
+import * as stylesImport from './GroupShowAll.scss';
+const styles: any = stylesImport;
+
+export class GroupShowAll extends BaseComponent<IGroupDividerProps, {}> {
+  public render(): JSX.Element | null {
+    let { group, groupLevel, showAllLinkText } = this.props;
+
+    if (group) {
```

Comment:
showAllLinkText  isn't required, so what happens if it is not supplied? We either need a default value or should hide the showAllLink unless text is supplied...

Reply:
I'm thinking we should go with default of "Show All", in which case we should update the interface with @default Show All, as well.